substrate/frame/transaction-storage/src/benchmarking.rs

```diff
@@ -129,7 +126,8 @@ benchmarks! {
 	store {
 		let l in 1 .. T::MaxTransactionSize::get();
 		let caller: T::AccountId = whitelisted_caller();
-		T::Currency::make_free_balance_be(&caller, BalanceOf::<T>::max_value());
+		let initial_balance = BalanceOf::<T>::max_value().checked_div(&2u32.into()).unwrap();
```

Comment:
We don't need a `checked_div` when we know we aren't dividing by 0

Reply:
this is just benchmark setup, no real point in optimizing.. I prefer to leave it like this (defensive even if useless)